src/reanimated2/interpolation.js

```diff
@@ -112,9 +114,7 @@ function internalInterpolate(x, l, r, ll, rr, type) {
 export function interpolate(x, input, output, type) {
   'worklet';
   if (x && x.__nodeID) {
-    throw new Error(
```

Comment:
I'd leave here console.warn.

Reply:
Like those ones?

Reply:
I'd go with something like:
"interpolate() was renamed to interpolateNode() in Reanimated 2. Please use interpolateNode() instead".